src/main/java/org/wso2/carbon/connector/googlefirebase/FirebaseConfig.java

```diff
@@ -80,19 +80,27 @@ public void connect(MessageContext messageContext) {
                     log.debug("Started to initialize the firebase admin sdk with the provided firebase service " +
                             "account's credentials");
                 }
+                log.info("Started to initialize the firebase admin sdk with the provided firebase service " +
+                        "account's credentials");
```

Comment:
Will this get printed for each requested or only at deployment ?

Reply:
Only upon the first request. That's how it is designed.